### PR TITLE
theme: add style to the possible matches UI

### PR DESCRIPTION
**Pull request by Dinika**
Signed-off-by: Dinika Saxena <dinika.saxena@cern.ch>

## Motivation and Context
Style the UI for possible matches added in inspirehep-search-js (PR [#65](https://github.com/inspirehep/inspirehep-search-js/pull/65))

On search results page (holdingpen)
screenshot from 2018-02-02 10-28-09

On the detailed record view:
screenshot from 2018-02-02 10-28-27

Success message:
screenshot from 2018-01-31 11-45-07

## Checklist:

- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [x] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by drjova** [comment]
@Dinika thanks :) could you please also upload a screenshot?

**Comment by Dinika** [comment]
@drjova I totally forgot about that. I've added the screenshot now. Also, I need to update this PR since it doesn't handle the 'None of these' case right now. I'll do that asap.

**Comment by michamos** [comment]
what about the detailed page? does it show the same thing there?

**Comment by Dinika** [comment]
@michamos I updated the PR. So, now the detailed record view shows the UI for possible matches.

**Comment by drjova** [comment]
@Dinika if we merge it before the backend, will it break?

**Comment by drjova** [comment]
ci merge please

File: inspirehep/modules/workflows/search.py [hunk 1]
```diff
@@ -43,7 +43,8 @@ def holdingpen_search_factory(self, search, **kwargs):
         'metadata.acquisition_source', 'metadata.arxiv_eprints',
         'metadata.arxiv_categories', '_workflow',
         '_extra_data.relevance_prediction', '_extra_data.user_action',
-        '_extra_data.classifier_results.complete_output', '_extra_data.journal_coverage'
+        '_extra_data.classifier_results.complete_output', '_extra_data.journal_coverage',
+        '_extra_data._action', '_extra_data.matches',
```
Comment (david-caro) [inline inspirehep/modules/workflows/search.py]: This change was expected? It looks alien to the PR

**Pull request merged**